assets/stylesheets/_base/_base.scss

```diff
@@ -30,11 +27,11 @@ blockquote {
 
 // Anchor
 a {
-  color: color(red--semi-dark);
+  color: palette(red, semi-dark);
```

Comment:
It's part of [flask](https://www.npmjs.com/package/flask). It's a great way to manage colors and fonts, IMO.